FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -354,9 +354,9 @@ - (BOOL)insertMetadataTableWithValues:(NSDictionary *)columnNameToValue {
   RCN_MUST_NOT_BE_MAIN_THREAD();
   static const char *SQL =
       "INSERT INTO " RCNTableNameMetadata
-      " (bundle_identifier, fetch_time, digest_per_ns, device_context, "
+      " (bundle_identifier, namespace, fetch_time, digest_per_ns, device_context, "
```

Comment:
what happen if user download a previous version of app using a previous version of remote config. Will this cause crash? 
Also once table has changed, will the old data be reset. Although I think metadata does not hold critical info so it's not as bad as the first concern.

Reply:
have we tried install previous version of remote config and then update to this new version? Just wanna make sure it still functions without crashes.

Reply:
Addressed in the earlier comment, but we are no longer updating the schema of an existing table.

It is true that metadata will initially be empty, but it doesn't affect previously fetched config values

Reply:
theoretically yes, that's why we should test  it out in real devices/simulators.